modules/persistent-storage-csi-gcp-filestore-wif.adoc

```diff
@@ -0,0 +1,126 @@
+= Preparing to install the {gcp-short} Filestore CSI Driver Operator with Workload Identity
+
+If you are planning to use {gcp-wid-short} with Google Compute Platform Filestore, you must obtain certain parameters that you will use during the installation of the {gcp-short} Filestore Container Storage Interface (CSI) Driver Operator.
+
+.Prerequisites
+* Access to the cluster as a user with the cluster-admin role.
+
+// Put note in install area of docs to remind users to take note of the identity pool ID and the provider ID
+
+.Procedure
+
+To prepare to install the {gcp-short} Filestore CSI Driver Operator with Workload Identity:
+
+. Obtain the project number:
+
+.. Obtain the project ID by running the following command:
++
+[source, terminal]
+----
+$ export PROJECT_ID=$(oc get infrastructure/cluster -o jsonpath='{.status.platformStatus.gcp.projectID}')
+----
+
+.. Obtain the project number, using the project ID, by running the following command:
++
+[source, terminal]
+----
+$ gcloud projects describe $PROJECT_ID --format="value(projectNumber)"
+----
+
+. Find the identity pool ID and the provider ID:
++
+During cluster installation, the names of these resources is provided to `ccoctl` with the `--name parameter`. See Step 3 of Section _Creating {gcp-short} resources with the Cloud Credential Operator utility_.
+
+. Create Workload Identity resources for the {gcp-short} Filestore Operator:
+
+.. Create a `CredentialsRequest` file using the following example file:
++
+.Example Credentials Request YAML file
+[source, YAML]
+----
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  name: openshift-gcp-filestore-csi-driver-operator
+  namespace: openshift-cloud-credential-operator
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+spec:
+  serviceAccountNames:
+  - gcp-filestore-csi-driver-operator
+  - gcp-filestore-csi-driver-controller-sa
+  secretRef:
+    name: gcp-filestore-cloud-credentials
+    namespace: openshift-cluster-csi-drivers
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+	kind: GCPProviderSpec
+    predefinedRoles:
+    - roles/file.editor
+    - roles/resourcemanager.tagUser
+    skipServiceCheck: true
+----
+
+.. Use the `CredentialsRequest` file to create a {gcp-short} service account by running the following command:
++
+[source, terminal]
+----
+$ ./ccoctl gcp create-service-accounts --name=<filestore-service-account> <1>
+--workload-identity-pool=<workload-identity-pool> <2> 
+--workload-identity-provider=<workload-identity-provider> <3> 
+--project=<project-id> <4> 
+--credentials-requests-dir=/tmp/credreq <5>
+----
+<1> <filestore-service-account> is a user-chosen name. 
+<2> <workload-identity-pool> comes from Step 2 above.
+<3> <workload-identity-provider> comes from Step 2 above.
+<4> <project-id> comes from Step 1.a above.
```

Comment:
You've just had them get these values, and called them by name in those steps. Can refer to the actual items and not reference steps that might change to avoid doc maintenance toil. 
```suggestion
<1> `<filestore-service-account>` is a user-chosen name. 
<2> `<workload-identity-pool>` is the identity pool ID.
<3> `<workload-identity-provider>` is the provider ID.
<4> `<project-id>` is the project ID.
```